vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/GraphQLHandlerImpl.java

```diff
@@ -304,8 +371,13 @@ private void executeOne(RoutingContext rc, GraphQLQuery query) {
   }
 
   private String getContentType(RoutingContext rc) {
-    String contentType = rc.parsedHeaders().contentType().value();
-    return contentType.isEmpty() ? "application/json" : contentType.toLowerCase();
+    String contentType = rc.request().headers().get(HttpHeaders.CONTENT_TYPE);
+    if (contentType == null || contentType.toLowerCase().contains("application/json")) {
+      return "application/json";
+    }
+
+    contentType = contentType.toLowerCase();
+    return contentType.contains("multipart/form-data") ? "multipart/form-data" : contentType;
```

Comment:
No. the header will return the boundary as well. I could substring it... but I don't know
if it will make sense.

Reply:
See my comment above. Master has an implementation of `getContentType` that works with multipart requests too.

Reply:
>Unless vertx is doing something under the hood (and I doubt it), we must remove the boundary from check

I actually does. See the latest implementation in master branch that uses `parsedHeaders`

Reply:
Please do not change the `getContentType` implementation that is currently in `master`. It works fine because it uses the `parsedHeaders` values form `vertx-web`.